test/language/directive-prologue/10.1.1-2gs.js

```diff
@@ -9,10 +9,10 @@ es5id: 10.1.1-2gs
 description: >
     Strict Mode - Use Strict Directive Prologue is ''use strict''
     which lost the last character ';'
-negative: NotEarlyError
-flags: [noStrict]
```

Comment:
The configuration `[raw, noStrict]` is a little redundant with the definition I'm proposing for `raw` (above):

> **`raw`** - execute the test without any modification (no helpers will be
>  available); necessary to test the behavior of directive prologue; implies
>  `noStrict`

We could do this to maintain backwards compatibility with the V8 runner, but since specifying both flags is not technically necessary, any future change could introduce a breaking test. Practically speaking, I don't think we expect too many new tests to need the `raw` flag, but it still seems safer to me for the V8 runner to update to understand the new semantics. Is that a feasible alternative? If so, I'd be happy to follow up with that change if/when this patch lands in `master`.